keyboards/handwired/uthol/keymaps/oled/rules.mk

```
@@ -0,0 +1,2 @@
+OLED_DRIVER_ENABLE = yes
```

This was changed to:
```suggestion
OLED_ENABLE = yes
```